CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 # TAMR AWS EMR module
 
+## v7.3.2 - April 11st, 2022
+* Adds example usage of EBS encryption in examples/static-spark-encrypted.
```

Comment:
There was a PR where we discussed not adjusting the version of a modules when there are only example updates.

Reply:
Fixed.